### DISCUSSION TITLE
今天，我开始自学了！ - Code Journey 编程之旅

### DISCUSSION THREAD

**Discussion by giscus** [Open-BJUT/CodeJourney#4] [Announcements]
# 今天，我开始自学了！ - Code Journey 编程之旅

https://open-bjut.github.io/CodeJourney/%E4%BB%8A%E5%A4%A9%EF%BC%8C%E6%88%91%E5%BC%80%E5%A7%8B%E8%87%AA%E5%AD%A6%E4%BA%86%EF%BC%81/

**Comment by Nagi-ovo** [comment]
建议“计算机科学与技术”或其它偏电子的专业的同学优先学习C\C++，不喜欢看书的话可以去b站找一门优质外国网课学习：）
